docs/source/client/actions/aliases.md

```diff
@@ -7,120 +7,59 @@ Handles transitive references (for example a -> b, b -> c becomes a -> c) and ig
 
 ## Get aliases list
 
-### Query function
+```{js:function} getAliasesListQuery
 
 Use the `getAliasesListQuery` function to get the query for fetching the aliases list.
 
-### Hook
-
-Use the `useGetAliasesList` hook to get the aliases list.
+:hook: `useGetAliasesList`
+```
 
 ## Get aliases
 
-### Query function
+```{js:function} getAliasesQuery(path)
 
 Use the `getAliasesQuery` function to get the query for fetching the aliases for a page.
 
-### Hook
-
-Use the `useGetAliases` hook to get the aliases for a page.
-
-### Parameters
-
-- **path**: string
-
-  - **Required:** Yes
+:arg string path: Description of the `path` parameter.
+:hook: `useGetAliases`
+```
 
 ## Add aliases for many pages
 
-### Mutation function
+```{js:function} createAliasesMutation(data)
```

Comment:
The next three went sideways. See example: https://deploy-preview-5576--volto.netlify.app/client/actions/aliases.html#add-aliases-for-many-pages

I'm not sure how to document the structure of an object as an argument. If you don't have any better idea, I'd try this (untested):

```
:arg array data: It can have the following fields:
  - items: An array of objects with the following fields:
    - path: string
    - redirect_to: string
    - [datetime]: string
```

You might need to indent the list items to the right amount. I don't know if Markdown for list items will even work in the value for arguments.

The ultimate fallback would be to convert the object into narrative text, which is not great.

Reply:
Okay, lets try it and see if it's better

Reply:
You can also do `make docs-livehtml` to preview docs in real-time. It's a lot faster than waiting for Netlify. It still doesn't look quite right. https://deploy-preview-5576--volto.netlify.app/client/actions/aliases.html

Reply:
@stevepiercy I tried doing `make docs-livehtml` but unfortunately I am getting this error : 
`/usr/bin/sh: line 2: bin/python: No such file or directory
make: *** [Makefile:99: bin/python] Error 127`

Reply:
Which way did you use to set up your environment for Volto docs? There are two ways, depending on the scope.

- [Documentation](https://6.docs.plone.org/volto/contributing/documentation.html) (volto only)
- [Building and checking the quality of documentation](https://6.docs.plone.org/contributing/documentation/setup-build.html) (Plone 6 docs, includes volto and other packages)

It looks like you do not have a Python virtual environment installed, according to this:

```console
/usr/bin/sh: line 2: bin/python: No such file or directory make: *** [Makefile:99: bin/python] Error 127
```

Line 99 in `Makefile` has this:

```make
	python3 -m venv . || virtualenv --clear --python=python3 .
```

which means "Create a Python 3 virtual environment in the current directory, if possible, else use `virtualenv` to wipe and recreate a Python 3 virtual environment." It fails because you either do not have Python installed on your system or, if you do have it installed, Python is not in your system `$PATH`.

If you do not have Python installed, see the advice to install [Python](https://6.docs.plone.org/contributing/documentation/setup-build.html#python). Else you will need to ensure that your `$PATH` is configured to point to a Python on your system. Please let me know.